implementations/rust/ockam/ockam_transport_core/src/transport.rs

```diff
@@ -5,6 +5,9 @@ use ockam_core::{async_trait, Address, Result, TransportType};
 /// At minimum, a Transport must be able
 ///  - return its type
 ///  - instantiate workers for all the addresses with that transport type in a Route
+
+pub const MAXIMUM_MESSAGE_LENGTH: usize = (u16::MAX - 2) as usize; //  - 2 bytes for length
```

Comment:
I've been reviewing this with the team and here we should define the general limit that is common to any transport, which should be `u16::MAX`.

Then, if a specific transport needs to restrict this limit, that should be done in that transport crate (e.g. tcp, udp). We'll create a different issue or this.

Reply:
separate PR, so for this PR no change needed?